microsoft-edge/toc.yml

```diff
@@ -1451,6 +1451,12 @@
                 - name: WebView
                   uid: 'WebView2Script!WebView:class'
 
+            - name: HoloLens 2
```

Comment:
Maybe put this under .NET in TOC if this is also under .NET tab set in microsoft-edge/webview2/webview2-api-reference.md?

Reply:
I agree. We have different supported UI frameworks under .NET and this should be like one of those, rather than at the level of C++,.NET,WinRT which are the different runtime/languages we support.

Reply:
Updated. I kept the HoloLens 2 sub hear. I can remove it, if you prefer.

Reply:
maybe this thread can be resolved